app/views/admin/project_logs/_log_list.html.slim

```diff
@@ -1,18 +1,21 @@
-
 - if @logs.empty?
   = content_tag(:div, t('common.no_records'), class: 'alert alert-info')
 - else
   .show-actions.hidden-print
     a data-expand-all=".logs-index" href="#" = t('common.expand_all')
-    a data-hide-all=".logs-index" href="#" = t('common.hide_all')
+    - if @step && @context && @context == "calendar"
+      a data-hide-all=".logs-index" href="#" = t('common.condense_all')
+    - else
+      a data-hide-all=".logs-index" href="#" = t('common.hide_all')
```

Comment:
The only thing changing here is the link text, yes? It would be better to use a ternary operator inside the `t` method call. That way you're not duplicating the rest of the code.